src/plugins/usage_collection/server/collector/usage_collector.ts

```diff
@@ -22,25 +22,32 @@ import { KIBANA_STATS_TYPE } from '../../common/constants';
 import { Collector, CollectorOptions } from './collector';
 
 // Enforce the `schema` property for UsageCollectors
-export type UsageCollectorOptions<T = unknown, U = T> = CollectorOptions<T, U> &
-  Required<Pick<CollectorOptions<T, U>, 'schema'>>;
+export type UsageCollectorOptions<
+  TFetchReturn = unknown,
```

Comment:
What's the purpose of changing the type names here? is it just to make `T, U` more descriptive?

Reply:
Yes, that is correct. I was getting a bit lost with so many generics 😓